srcDLL/op2ext.h

```diff
@@ -61,12 +61,12 @@ OP2EXT_API void Log(const char* message);
 
 
 // Performs a case insensitive search of loaded module names, returning true if found.
-// A console module's name is the name of the directory the console module is stored in.
+// A console module's name is the relative directory passed in via the /loadmod command.
```

Comment:
When stating "relative directory" it's often helpful to state what it is relative to. It could be the current working directory, or the executable's directory. Often the two are the same, but not always. Ex:
"A console module's name is the folder name parameter passed in via the /loadmod command. The folder name should be relative to the executable's folder, with no trailing slash"